packages/host/app/components/operator-mode/stack.gts

```diff
@@ -37,6 +38,11 @@ export default class OperatorModeStack extends Component<Signature> {
     for (let i = this.args.stackItems.length - 1; i > itemIndex; i--) {
       itemsToDismiss.push(this.args.stackItems[i]);
     }
+
+    // do closing animation on last item
+    const lastItem = this.args.stackItems[this.args.stackItems.length - 1];
+    await this.args.publicAPI.doCloseAnimation(lastItem.card);
```

Comment:
it would be simpler to just pass in the `stackItem`, instead of passing in the card for the stack item, and then trying to figure out which stack item has that card on the other end.